### ISSUE TITLE
[Feature Request] Mark as Read Button on Post

### ISSUE THREAD

**Issue by DarkArc** [Xyphyn/photon#399]
**Is your feature request related to a problem? Please describe.**
Currently, when scrolling through posts you have to go to the "More Actions" menu to access the "Mark as Read" option.

**Describe the solution you'd like**
As someone that uses the "Hide Read Posts" (server side) option, I'd like to add the "Mark as Read" to the actions that are visible on the post itself (currently only moderation actions and the save button).

This would allow me to quickly "acknowledge" a post I've seen but I'm not interested with engaging with further (currently my options are to actually open the post or to open the "More Actions" menu and find the button; and it's just a little more friction than I'd like).

**Describe alternatives you've considered**
It might be useful if you could just generally pick what action are in the "More Actions" menu vs on the post visibly.

**Comment by Xyphyn** [comment]
What if I made right clicking on the title mark a post as read? I want to keep the action bar clutter free.

**Comment by DarkArc** [comment]
Hm, I think that might be a bit too subtle(?)

Maybe if you did something like you did for the "Click a pin to send it to the top of the sidebar. Right click a pinned item to remove it."

That way folks can choose the items that are most valuable to them to be shown? As an example, personally, moderation actions and saving a post are things I use very infrequently (so I don't need them there ... where as mark as read would be a very valuable button).

**Comment by enpinzolas** [comment]
Agreed on that something needs to be done to mark posts as read more easy. Both for people that like to hide posts as they scroll (mark as read on scroll would be awesome too) and for those that want to just mark them as read.

**Comment by DarkSpectrum** [comment]
My vote would be marking as read by scrolling alone as the most ideal solution. I really like Photon but get annoyed every time I go to the main page and see the same content. Thanks for your hard work! :)

**Comment by sevonj** [comment]
> marking as read by scrolling alone

If that gets implemented, it should be behind a toggle and opt-in.
